ibm/resource_ibm_is_instance.go

```diff
@@ -451,6 +453,100 @@ func resourceIBMISInstance() *schema.Resource {
 				Type:        schema.TypeInt,
 				Optional:    true,
 			},
+			isInstanceDedicatedHost: &schema.Schema{
```

Comment:
actually scoped spec has a response dedicated_host in Post and GET calls since long time. i thought it will be merged to production but in production it is not available. so i kept it commented. But anyway i have removed it now from all places. will add later if it comes to production